### DISCUSSION TITLE
PIC32 Lead Times

### DISCUSSION THREAD

**Discussion by JacobChrist** [chipKIT32/chipKIT-core#480] [Q&A]
Anyone know why Microchip lead times for PIC32's are all out at January 2023?  I've been looking at lead times for the last two weeks and it seems like the Microchip Direct web page is just making the lead times the current day + 1 year and 1 week.  Does anyone have an real information?

**Comment by EmbeddedMan** [comment]
It may be that for some parts they just can't give any sort of realistic estimate. I've seen other manufacturers do that. Because of how crazy things are right now they can't even give a real estimate. So it's just "way the heck out there somewhere". Lots of folks are just placing orders, and hoping that the get some product trickling into their warehouse and then they do builds based on what parts they have on hand. Of course this makes the semiconductor manufacturer's job even harder. It's a positive feedback loops system right now I think.

**Comment by majenkotech** [comment]
The problem is that they bought Atmel. Atmel are heavily used by the
automotive industry, and keeping up with their demand is swamping the
microchip fabs. Microchip should never have bought Atmel...

**Comment by JacobChrist** [comment]
In my other life, we use ST parts and they have the same crunch in the same boat.  I personally have though about buying a reel of $0.09 3.3V regulators because I need 50 and wanted to make sure I have some if I need 51.  I'm sure I'm not the only one with this thought.  Also in my other life we have bought a five year supply of parts months ago because we don't when the end is in sight, if we are not alone there will probably be a glut of parts on the other end of this mess.

**Comment by majenkotech** [comment]
Last time I bought from Microchip Direct the website stated one lead time, then the order confirmation gave me another date. I ordered 11th October 2021 for a purported 9th November 2021 shipping date. The confirmation came through with a 29th June 2022 shipping date...!

Two weeks later the chips shipped and arrived before the 9th Nov estimated date...

So don't believe the dates on the website, they're really only a worst-case scenario and there to temper your expectations.
